src/index.tsx

```diff
@@ -2,12 +2,18 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './components/app/app.tsx';
 
+import {offers} from 'mocks/offers.ts';
+import {reviews} from 'mocks/reviews.ts';
+
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
 
 root.render(
   <React.StrictMode>
-    <App />
+    <App
+      offers = {offers}
+      reviews = {reviews}
+    />
```

Comment:
форматируем только так
```suggestion
    <App
      offers={offers}
      reviews={reviews}
    />
```